src/pages/code-search.tsx

```diff
@@ -1,67 +1,59 @@
-    {
-        title: 'Code search tidbits: 4 underrated features',
-        description:
-            "You can do some pretty wild things with Sourcegraph that you won't find in any other code search tool today. Read about 4 short-and-sweet tidbits of underrated search features that go a bit extra.",
-        type: 'Blog',
-        href: '/blog/code-search-tidbits-episode-1',
-    },
-    {
-        title: 'How we used Notebooks to make our CI more accessible and understandable',
-        description:
-            'Learn how Notebooks—Sourcegraph’s in-app living documentaiton—makes it easier to document complex codebases.',
-        type: 'Blog post',
-        href: '/blog/notebooks-ci',
-        img: {
-            src: '/blog/thumbnails/notebooks-ci.jpg',
-            alt: 'Sourcegraph Noebooks CI',
-        },
-    },
-    {
-        title: 'Key traits of a code intelligence platform',
-        description:
-            'Sourcegraph is more than search. Engage and enable your teams by helping developers get unblocked, resolve issues faster, and gain codebase insights.',
-        type: 'Guide',
-        href: '/guides/key-traits-of-a-code-intelligence-platform.pdf',
-    },
-]
 
 export const CodeSearchPage: FunctionComponent = () => (
     <Layout
         meta={{
-            title: 'Sourcegraph - Code Search',
+            title: 'Sourcegraph | Code Search',
             description:
-                'Onboard to a new codebase, find answers faster, and identify security risks with Sourcegraph Code Search. Search across all the repositories you work with.',
+                'Code Search helps devs explore their codebase, make large-scale migrations, and fix security issues faster—especially in large, distributed codebases.',
+            image: 'https://about.sourcegraph.com/code-search/code-search-og.png',
         }}
         hero={
-            <Hero
-                variant="lightNebulousMars"
-                product="code search"
-                title={'Search your code.\nAll of it.'}
-                subtitle="Onboard to a new codebase, understand code faster, and identify security risks with universal code search."
-                cta={<StandardCallToAction buttonLocation={buttonLocation.hero} />}
-                displayUnderNav={true}
-            />
+            <>
+                <ContentSection
+                    parentClassName="!py-0 !px-sm overflow-x-clip"
+                    className="grid grid-cols-1 pb-8 pt-20 md:grid-cols-2 md:pt-28 md:pb-16"
+                >
+                    <div className="flex w-full flex-col">
+                        <Heading size="h1" className="text-white md:text-start md:!text-[52px]">
+                            Find, understand, and fix your code
+                        </Heading>
+
+                        <p className="pt-6 pb-5 text-3xl text-gray-200">
+                            Code Search, along with complimentary tools, helps devs find, fix, and onboard to new code
```

Comment:
`complimentary` means free. `complementary` means what you intend.

```suggestion
                            Code Search, along with complementary tools, helps devs find, fix, and onboard to new code
```